client-core/src/service/key_service.rs

```diff
@@ -21,32 +21,32 @@ where
         KeyService { storage }
     }
 
-    /// Adds a new public-private keypair to storage
-    pub fn add_keypair(
+    /// Adds a new wallet_name-private keypair to storage
+    pub fn add_wallet_private_key(
```

Comment:
some explanation?
what's this privatekey?

Reply:
the private key is the one when a wallet created and the view key comes from this private key

Reply:
if it's just one wallet, set_wallet_viewkey_private?